components/Footer.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { SiInstagram, SiFacebook, SiLinkedin, SiYoutube, SiGithub } from "react-icons/si";
+
+const footerContainer = "flex flex-col items-center justify-center text-[#A689FF] h-36 md:text-[#FFFFFF] md:bg-[#7055FD]";
+const iconContainer = "flex flex-row space-x-8";
+const iconsStyle = "text-[#7055FD] text-3xl md:text-[#FFFFFF] md:cursor-pointer md:text-4xl";
+
+const icons = [
+  { name: "facebook", link: "https://www.facebook.com/CodeTheChangeYYC", logo: <SiFacebook /> },
+  { name: "instagram", link: "https://www.instagram.com/codethechangeyyc/", logo: <SiInstagram /> },
+  { name: "linkedin", link: "https://www.linkedin.com/company/codethechangeyyc/", logo: <SiLinkedin /> },
+  { name: "youtube", link: "https://www.youtube.com/channel/UC4wZt-bCL31HjxUF-zc5U_g", logo: <SiYoutube /> },
+  { name: "github", link: "https://github.com/Code-the-Change-YYC", logo: <SiGithub /> },
+];
+
+const handleClick = (link) => {
+	window.open(link);
```

Comment:
I think you can use the `Link` component here from next instead of handling click yourself. No need to make a change tho because you can re-use the `SocialMediaIcons` component from the other PR #14

Reply:
Just gonna wait for response to the comment earlier before moving on with any changes